src/main/java/com/endava/internship/warmup/domain/service/ArrayProcessorWithForLoops.java

```diff
@@ -148,6 +306,12 @@ public void validateForMatrixMultiplication(int[][] leftMatrix, int[][] rightMat
      */
     @Override
     public int[] distinct(final int[] input) {
-        throw new UnsupportedOperationException("Method not implemented yet");
+        Set<Integer> s = Arrays.stream(input)
```

Comment:
There is a method distinct() that you can use over a stream to get distinct values.